streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -69,7 +68,6 @@
     private final ChangelogReader changelogReader;
     private final UUID processId;
     private final String logPrefix;
-    private final StreamsMetricsImpl streamsMetrics;
```

Comment:
My IDEA pointed out that this was never used

Reply:
You're right, I was looking at another branch (kafka-raft) again.. the jumping back-and-forth reviewing those PR keep confusing me, sorry.

Reply:
Lol, Guozhang, that's like the 3rd time in a week you've been looking at the old branch. I guess kafka-raft is not so up-to-date huh 😛

Reply:
No it is not at all :) https://github.com/confluentinc/kafka/commits/kafka-raft